source/reference/methods.txt

```diff
@@ -876,3 +876,90 @@ Role Management Methods
    * - :method:`db.updateRole()`
 
      - Updates a user-defined role.
+
+Replication Methods 
+-------------------
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+
+     - Description
+ 
+   * - ``rs.printSecondaryReplicationInfo()``
+
+     - Returns the status of the secondary members of the replica set. This 
+       is identical to the ``db.printSecondaryReplicationInfo()`` method. 
+       The server :method:`rs.printSlaveReplicationInfo()` method is referred 
+       to as ``rs.printSecondaryReplicationInfo()`` in ``mongosh``.
```

Comment:
So I don't think this blurb is entirely correct. I don't think we want to contrast "server" methods and "mongosh" methods. :method:rs.printSlaveReplicationInfo() is a _legacy shell_ method, and we cant to contrast that with a _mongosh_ method. Also, 

Consider something like the following:

```
This is identical to the ``db.printSecondaryReplicationInfo()`` method. 
This method returns the same output as the :method:`rs.printSlaveReplicationInfo()`
method in the legacy binary:`~mongo` shell.
```

We may want to be more direct and explicitly mention something like

```
:method:`rs.printSlaveReplicationInfo()` is deprecated in mongosh.
Use :method:`rs.printSecondaryReplicationInfo()` instead.
```